src/components/Catalog.vue

```diff
@@ -1,13 +1,13 @@
 <template>
-  <b-card no-body :class="classes" v-b-visible.400="load" :img-right="isList">
-    <b-card-img-lazy v-if="hasImage" class="thumbnail" offset="200" v-bind="thumbnail" />
+  <b-card no-body :class="classes" v-b-visible.400="load">
+    <b-card-img v-if="hasImage" v-bind="thumbnail" lazy />
```

Comment:
Where did the img-right, class and offset go?

Reply:
Okay, I think the offset had a meaning for lazy loading, so we still need to check for an alternative. Please add it to the todo list (where can I find it?)